x/poe/bootstrap.go

```diff
@@ -280,6 +263,34 @@ func BootstrapPoEContracts(ctx sdk.Context, k wasmtypes.ContractOpsKeeper, tk tw
 	}
 	logger.Info("validator voting contract", "address", validatorVotingContractAddr, "code_id", validatorVotingCodeID)
 
+	// setup trusted circle for ap
+	apTrustedCircleInitMsg := newAPTrustedCircleInitMsg(gs)
+	apDeposit := sdk.NewCoins(gs.ArbiterPoolContractConfig.EscrowAmount)
+	firstAPMember, err := sdk.AccAddressFromBech32(gs.ArbiterPoolMembers[0])
+	if err != nil {
+		return sdkerrors.Wrap(err, "first ap member")
+	}
+
+	apContractAddr, _, err := k.Instantiate(ctx, trustedCircleCodeID, firstAPMember, systemAdminAddr, mustMarshalJson(apTrustedCircleInitMsg), "arbter_pool", apDeposit)
+	if err != nil {
+		return sdkerrors.Wrap(err, "instantiate tg trusted circle contract")
+	}
+	poeKeeper.SetPoEContractAddress(ctx, types.PoEContractTypeArbiterPool, apContractAddr)
+	if len(gs.ArbiterPoolMembers) > 1 {
+		err = addToTrustedCircle(ctx, apContractAddr, tk, gs.ArbiterPoolMembers[1:], firstAPMember, gs.ArbiterPoolContractConfig.EscrowAmount)
+		if err != nil {
+			return err
+		}
+	}
+
+	// setup arbiter pool
+	apCodeID, err := k.Create(ctx, systemAdminAddr, tgArbiterPool, &wasmtypes.AllowEverybody)
+	if err != nil {
+		return sdkerrors.Wrap(err, "store tg ap contract: ")
+	}
+
+	logger.Info("arbiter pool contract stored", "code_id", apCodeID)
+
```

Comment:
Please pass the `apContractAddr` and the new `apVotingContractAddr` to the `setAllPoEContractsInstanceMigrators` call in L297

Reply:
Sorry, this is not correct. All PoE contracts are fetched by `types.PoEContractType_value` so this is done already.